### PR TITLE
Have ADR tools in a docker image

### PR DESCRIPTION
**Pull request by luszak**

### CODE REVIEW

**Comment by luszak** [comment]
@ClearcodeHQ/pyziomki please review

File: magefiles/adr/magefile.go [hunk 1]
```diff
@@ -1,46 +1,109 @@
...
-	return nil
+	command := strings.Split(buf.String(), " ")
+	return command, nil
+
 }
 
-// Init initializes adr in a given directory. Requires one argument - directory
-func Init(dir string) error {
-	return sh.RunV("adr", "init", dir)
+// Init creates ADR directory.
```
Comment (fizyk) [inline magefiles/adr/magefile.go:90]: describe used params as well

**Review by fizyk — CHANGES_REQUESTED** [review]

File: magefiles/adr/magefile.go (hunk 1 above)
Comment (luszak) [reply magefiles/adr/magefile.go]: done

**Review by pabramowicz — APPROVED** [review]

File: magefiles/adr/magefile.go [hunk 2]
```diff
@@ -1,46 +1,113 @@
...
+	}
+	return sh.Run(command[0], command[1:]...)
+
+}
+
+// New creates a new ADR entry.
+// By default adrDir, adrConfigDir, and adrTemplate are set.
+// If you wish to override them use ctx.WithValue() to
+// set corresponding values and pass the context to this function.
+//   * adrDir - directory which docker mounts as /docs
```
Comment (fizyk) [inline magefiles/adr/magefile.go:103]: and command's params ;)

**Review by fizyk — APPROVED** [review]

**Pull request merged**